configdb/hardware/api_views.py

```diff
@@ -72,14 +121,18 @@ class Meta:
 
     def state_filter(self, queryset, name, value):
         ''' Allows us to do queries like ?state=MANUAL instead of ?state=10 '''
-        print(name)
         for state in Instrument.STATE_CHOICES:
             if value.upper() == state[1]:
                 return queryset.filter(state=state[0])
         return queryset
 
 
 class InstrumentViewSet(FilterableViewSet):
+    custom_filter_annotations=[{'name': 'science_cameras', 'description': 'Set of science camera codes on the instrument',
+                                'type': 'Array of strings'},
+                               {'name': 'autoguider_camera', 'description': 'Autoguider code for the autoguider camera on the instrument'}]
```

Comment:
This is an example where the InstrumentFilter includes a couple of fields (`science_cameras`, `autoguider_camera`) in the `fields` Meta attribute of the filterset, but doesn't include them as explicit filters. Here they are being overridden with some better descriptions and type annotations.